Gopkg.toml

```diff
@@ -39,7 +39,7 @@ required = [
 [[constraint]]
   name = "github.com/percona/percona-toolkit"
   # version = "3.1.1"
-  branch = "PMM-4192_Mongo_explain_fails_fixing_tests"
+  branch = "PMM-6256_Explain_doesnt_work_2"
```

Comment:
👀

Reply:
There is new code in toolkit for this.

Reply:
What @askomorokhov means is that we should not merge the change with that branch

Reply:
https://github.com/percona/percona-toolkit/pull/459 was merged, so we should update a branch